homeassistant/components/zha/core/device.py

```diff
@@ -284,8 +284,12 @@ def _check_available(self, *_):
                         )
                         if not self._channels.pools:
                             return
-                        pool = self._channels.pools[0]
-                        basic_ch = pool.all_channels[f"{pool.id}:0"]
+                        try:
+                            pool = self._channels.pools[0]
+                            basic_ch = pool.all_channels[f"{pool.id}:0x0000"]
```

Comment:
Could be a const but not necessary

Reply:
I wanna get away from that channel ID format and will make it a const then